examples/notebooks/integrate_3rd_transforms.ipynb

```diff
@@ -0,0 +1,294 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Integrate 3rd party transforms into MONAI program"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how to integrate 3rd party transforms into MONAI program.  \n",
+    "Mainly shows transforms from `BatchGenerator`, `TorchIO` and `Rising`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install batchgenerators"
```

Comment:
perhaps with a specific version number is easier for us maintaining

Reply:
updated in the latest commit.
Thanks.